var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -130,17 +133,15 @@ class Sundials(CMakePackage):
     # External libraries incompatible with single precision
     conflicts('+klu',   when='precision=single')
     conflicts('+hypre', when='+hypre@:2.12.0 precision=single')
+    conflicts('+superlu-dist', when='precision=single')
 
     # External libraries incompatible with extended (quad) precision
     conflicts('+lapack',     when='precision=extended')
     conflicts('+superlu-mt', when='precision=extended')
+    conflicts('+superlu-dist', when='precision=extended')
```

Comment:
Need to set the the CMake variables to build the SuperLU_DIST module.

Reply:
Oops, good catch. Should be taken care of now.